Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM php:8.2-fpm
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG GITHUB_OAUTH_TOKEN
+ARG XDEBUG_VERSION="xdebug-3.3.2"
+
+ENV NVM_DIR /usr/local/nvm
+ENV COMPOSER_ALLOW_SUPERUSER=1
+ENV GITHUB_OAUTH_TOKEN=$GITHUB_OAUTH_TOKEN
+ENV PHP_DIR /usr/local/etc/php
+
+# base packages
+RUN apt-get update && apt-get install -y \
+    git \
+    curl \
+    libpng-dev \
+    libonig-dev \
+    libxml2-dev \
+    zip \
+    unzip \
+    redis-tools \
+    nano \
+    python3 \
+    make \
+    g++\
+    gpg \
+    gettext
+
+RUN apt clean && rm -rf /var/lib/apt/lists/*
+
+RUN docker-php-ext-install pdo_mysql mbstring exif pcntl bcmath sockets gettext
```

Comment:
please check missing exts
 "ext-gd": "*",
        "ext-imagick": "*",
        "ext-json": "*",
        "ext-openssl": "*",
        "ext-pdo": "*",
        "ext-zlib": "*",
from composer.json

Reply:
All the missing exts were added except for:

- ext-json: no longer needed in this version of PHP

- ext-openssl: adding this extension breaks the build due to this error: "Cannot find config.m4. Make sure that you run '/usr/local/bin/phpize' in the top-level source directory of the module". 
We´re gonna need to dig deeper here.

Reply:
i am gonna re open this
bc its not a feasible solution
we need to know why ext openssl is failing

Reply:
This is done